### DISCUSSION TITLE
购买了一年的专业版蓝灯，邮箱收不到验证吗，手机无法登录，请解决！！！

### DISCUSSION THREAD

**Discussion by ezeanya626** [getlantern/lantern-forum-cn#1084] [General]
邮箱：pallintimofei@gmail.com
![支付图片](https://github.com/user-attachments/assets/ed48a196-a77a-4be4-901f-63742d740fed)

**Comment by ezeanya626** [comment]
我是看到了产品说可以链接三个用户，我才购买使用的，请解决我的安卓端登录的问题，否则就退款！！

**Comment by MaxMKMax** [comment]
@ezeanya626 您好, 我已经帮您提交问题上去了. 请您等待几天. 我们会处理好的. 请您放心, 谢谢!
